projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts

```diff
@@ -71,6 +73,9 @@ export class NgxAuthFirebaseuiRegisterComponent implements OnInit, OnDestroy {
   @Input() passwordConfirmationText = 'Password Confirmation';
   @Input() passwordConfirmationErrorRequiredText = 'Password confirmation is required';
   @Input() passwordErrorMatchText = 'Password must match';
+  // crunck78 - request feature password length control for standalone registration component 
```

Comment:
i would suggest to remove the comment as there is no added value.

Reply:
Removed.